src/main/java/com/epam/training/designpatterns/fastfoodrestaurant/food/Food.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.training.designpatterns.fastfoodrestaurant.food;
+
+
+public interface Food {
+	
+	int applyEffect(int happiness);
```

Comment:
Nagyon jó. Bemegy a régi érték, kijön az új, módosított érték. A mögöttes logikát pedig elrejti az étel. Akár az aktuális `Client` példányt is megkaphatná ez a metódus és módosíthatná a `hapiness` értékét.